e2e/_suites/ingest-manager/README.md

```diff
@@ -52,7 +52,8 @@ This is an example of the optional configuration:
    # (Fleet mode) This environment variable will use the snapshots produced by Beats CI. If the above variable
    # is set, this variable will take no effect
    export ELASTIC_AGENT_USE_CI_SNAPSHOTS="true"
-   # (Stand-Alone mode) This environment variable will use the its value as the Docker tag produced by Beats CI.
+   # (Stand-Alone mode) This environment variable will use the its value as the Docker tag produced by Beats CI (Please look up here: https://container-library.elastic.co/r/observability-ci/elastic-agent)
+   export ELASTIC_AGENT_STAND_ALONE_VERSION="pr-20356"
    export ELASTIC_AGENT_STAND_ALONE_VERSION="78a762c76080aafa34c52386341b590dac24e2df"
    ```
```

Comment:
It's duplicated?

Reply:
Let me make it even clearer in the comments, something like _here you have two examples_ or similar